### ISSUE TITLE
Install Docker CE docs for Linux Mint need to be updated.

### ISSUE THREAD

**Issue by agirorn** [docker/docker.github.io#6332]
File: [install/linux/docker-ce/ubuntu.md](https://docs.docker.com/install/linux/docker-ce/ubuntu/), CC @gbarr01 

The section on install on Linux mint need to be updated. It does not work on Linux mint sylvia.
I got it working by using xenial instead of sylvia.

*SET UP THE REPOSITORY*

*step 4.*

```shell
sudo add-apt-repository \
   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
   xenial \
   stable"
```

**Comment by thaJeztah** [comment]
Yes; I agree we should remove those notes, because those distros are not tested, and notes can go outdated easily.
